include/blmc_robots/slider.hpp

```diff
@@ -2,23 +2,22 @@
  * \file slider.hpp
  * \author Manuel Wuthrich
  * \date 2018
- * \copyright Copyright (c) 2019, New York University and Max Planck Gesellschaft.
+ * \copyright Copyright (c) 2019, New York University and Max Planck
+ Gesellschaft.
```

Comment:
missing "*" I winder how the doc will react, I think it should be ok but I do not know

Reply:
I don't think that it should be an issue for the doc generation, but it doesn't look nice, so I'll fix it in any case.

Reply:
When I replace the "and" with "&" it just fits in one line.  I suggest we change it to & everywhere, at least in new files or when we notice it in a file we edit anyway.

Reply:
done